firebase.json

```diff
@@ -1,6 +1,6 @@
 {
   "hosting": {
-    "predeploy": "flutter clean; flutter build web",
+    "predeploy": "flutter clean flutter build web",
```

Comment:
I'm not sure this fixes anything. Removing the `;` results in a command that shouldn't do anything (or maybe just a flutter clean?)

Have you checked the contents of the build/web directory after this change?

Reply:
And another thing it never gets to build for the web because it gets stuck at trying to clean

Reply:
@k1ycee what platform are you using to do this codelab?